RunTests.ps1

```diff
@@ -42,11 +43,13 @@ if (-not $target_versions) {
     throw "Could not find any valid versions of Visual Studio installed"
 }
 
+$nodejstoolsroot = $PSScriptRoot
```

Comment:
oops, looks like we're duplicating logic - either use $rootDir or $nodejstoolsroot